src/main/java/io/cdap/delta/bigquery/Constants.java

```diff
@@ -26,6 +26,8 @@ public final class Constants {
   public static final String ROW_ID = "_row_id";
   public static final String OPERATION = "_op";
   public static final String BATCH_ID = "_batch_id";
+  public static final String SORT_KEYS = "_sort";
```

Comment:
nit: _sort -> _sort_keys

Reply:
I wanted to keep is the struct field as `_sort` and the actual nested key as `_key_0` (column - `_sort._key_0`), wdyt?

Reply:
I see what you meant , sg.